Anlab.Core/Models/AccountModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Anlab.Jobs.MoneyMovement
+{
+    public class AccountModel
+    {
+        public AccountModel(string rawAccount)
+        {
+            var temp = rawAccount.Split('-');

```

Comment:
@jSylvestre might be worth throwing in a regex or at least checking that temp has length 2 before doing the temp[1] index, otherwise this will throw an exception on a bad account instead of just returning null account info.

Reply:
@jSylvestre I know regex is no fun but with capture groups this whole thing could be one line w/ the regex, which would also do your length validation at the same time.   something like `(\w)-(\w{7})\/?(\w{5})?` which is basically 3 capture groups, one for chart, one for 7 length account, then an optional slash and an optional group for subacct.  Basically just plug in and use Regex.Matches to take care of it